README.md

```diff
@@ -1,3 +1,25 @@
 # Blackhole.service for DEIS
 
-This will be a repository for dockerized blackhole service for DEIS.
+### Blackhole.service
+
+An example file for fleetctl unit.
+
+```
+  [Unit]
+  Description=Turn on automated addition of blackhole routes for sshd attackers
+
+  [Service]
+  TimeoutStartSec=0
+  ExecStartPre=-/usr/bin/docker kill deis-blackhole
+  ExecStartPre=-/usr/bin/docker rm deis-blackhole
+  ExecStartPre=/usr/bin/docker pull khelben/deis-blackhole
```

Comment:
I need to move this image to some company account if this will go for production.